docs/pages/deploy-a-cluster/deployments/gcp.mdx

```diff
@@ -16,36 +16,20 @@ Teleport Cloud manages for you.
 
 We have split this guide into:
 
-- [Teleport on GCP FAQ](#teleport-on-gcp-faq)
 - [GCP Teleport Introduction](#gcp-teleport-introduction)
 - [GCP Quickstart](#gcp-quickstart)
 
 (!docs/pages/includes/cloud/call-to-action.mdx!)
 
-## Teleport on GCP FAQ
```

Comment:
I didn't find this section compelling enough to keep. If someone's reading our GCP docs we probably don't need to sell them on Teleport.